### ISSUE TITLE
Polishing fitting functionality

### ISSUE THREAD

**Issue by mkhorton** [materialsintelligence/propnet#248]
* Why does minimum score 0 not work?
* Be able to supply scores by default in Graph construction
* Think about how to report error in aggregated quantities

**Comment by clegaspi** [comment]
Regarding the minimum score of 0, that was a constraint Joey added, that the minimum must be 5% weight. @montoyjh was there a specific reason?

**Comment by mkhorton** [comment]
Yes, I understand the default is 0.05, but if you set it to 0 the fitting just doesn't work.
